### ISSUE TITLE
Osquery on Windows Server 2019

### ISSUE THREAD

**Issue by noale1** [osquery/osquery#6342]
Hi!
I  have been trying to download Osquery for one of my servers - Windows Server 2019.
At the beginning I downloaded the latest version - 4.2.0 and I thought it installed successfully, but unfortunately later I found out that the folder "C:\Program Files\osquery" was missing several files and folders. 
It contained only the "osquery.db" folder and "osqueryd.pidfile", it was missing "osqueryd" itself and the rest of the files and folders that are supposed to appear.
I thought maybe it was a problem with the specific version because I have never used it so I don't have experience with it. 
I tried version 4.1.2 on a windows 10 computer and it worked perfectly fine! So I tried that version on my server as well, and the exact same thing happened. I figured it was a problem with my server and I tried looking up Osquery for Windows servers and couldn't find an answer.

I would really appreciate it if someone has an answer as to why this happens!!
Thanks!

**Comment by Smjert** [comment]
Hi @noale1!
That's odd, the MSI we release should work on Windows Server 2019; maybe it's a problem of permissions of the user installing the MSI?

How exactly are you installing it? Through command line or UI?
If through command line, have you tried via UI?

Could you try to add this option when launching the installation from CLI
```
/l*v osquery_installtion_log.txt
```
so that we can maybe see if there are errors to help us investigate this issue?

**Comment by noale1** [comment]
Hey,
So I ended up figuring it out. I have a mounted disk on my server, and for some reason, even though I ran it from C: it installed the entire package in B: (the other disk). 
I'll try to just move the files that are missing from B: to C: and change the path for the service to run from.

It's still pretty odd because it did create the folder in C:\Program Files\osquery that contained what I mentioned before ("osquery.db" folder and "osqueryd.pidfile")
And B:\Program Files\osquery contained the **rest** of the files and folders that are supposed to appear in the osquery folder.

Here are the screenshots -

C:
![C](https://user-images.githubusercontent.com/62059754/78137807-94ae7400-742e-11ea-9146-9d21916344b6.PNG)

B:
![B](https://user-images.githubusercontent.com/62059754/78137815-97a96480-742e-11ea-8481-ff73d2ffaeb6.PNG)

I'll also attach the installation log incase you still want to investigate it.
[osquery_install_log.txt](https://github.com/osquery/osquery/files/4415173/osquery_install_log.txt)


Thanks!!
